mergify_engine/prom_exporter.py

```diff
@@ -134,6 +137,11 @@ def collect_metrics():
             if e.status not in (403, 401) and e.status < 500:
                 raise
 
+    with futures.ThreadPoolExecutor() as executor:
```

Comment:
I think the default is the number of CPU, though in this case it's unlikely to be a good default.

Reply:
Number of cpu x5